common/components/internal-header/template.njk

```diff
@@ -54,14 +49,18 @@
     {% endif %}
 
     {% if params.navigation %}
-    <button type="button" role="button" class="app-header__menu-button js-header-toggle" aria-controls="navigation" aria-label="Show or hide Top Level Navigation">Menu</button>
     <nav>
       <ul id="navigation" class="app-header__navigation {{ params.navigationClasses if params.navigationClasses }}" aria-label="Top Level Navigation">
         {% for item in params.navigation %}
           {% if item.href and item.text %}
             <li class="app-header__navigation-item{{ ' app-header__navigation-item--active' if item.active }}">
               <a class="app-header__link" href="{{ item.href }}"{% for attribute, value in item.attributes %} {{attribute}}="{{value}}"{% endfor %}>
                 {{ item.text }}
+                {% if item.subtext %}
+                  <span class="app-header__link__sub-text">
```

Comment:
I think this should be:
```suggestion
                  <span class="app-header__link-sub-text">
```

to better fit with the [BEM](http://getbem.com/) naming convention.